inttest/common/footloosesuite.go

```diff
@@ -737,7 +739,7 @@ func (s *FootlooseSuite) createConfig() config.Config {
 		{
 			Type:        "bind",
 			Source:      binPath,
-			Destination: "/usr/bin/k0s",
+			Destination: defaultK0sBinPath,
```

Comment:
I think we should use `s.K0sFullPath` here so we can override the default path to k0s in individual tests.